Resources/Private/JavaScript/asset-variants/src/components/Variant.tsx

```diff
@@ -54,10 +56,18 @@ const Variant: React.FC<VariantProps> = ({
     width,
     height,
     previewUrl,
+    id,
+    hasCrop,
 }: VariantProps) => {
-    const classes = useStyles();
+    // TODO: Find out why we need to check both
```

Comment:
Are you wondering why there can exist variants without an preset identifier in a Neos installation?

Reply:
I guess thats because custom crops (e.g. in Fusion) are also stored as variants and have no preset identifier? 

I took that code from the current media browser version, and I am wondering why a variant with preset identifier should not be croppable, and more important: why does checking for an existing crop area ensure that the image is croppable. If I have a 16:9 image and a 16:9 preset, technically there is no need for a crop area over the whole image, but I should still be able to choose a smaller area